examples/secure_cloud_function_with_sql/main.tf

```diff
@@ -66,7 +67,7 @@ module "secure_harness" {
   security_project_extra_apis = ["secretmanager.googleapis.com"]
 
   serverless_project_extra_apis = {
-    "prj-secure-cloud-function" = ["servicenetworking.googleapis.com", "sqladmin.googleapis.com", "cloudscheduler.googleapis.com", "networksecurity.googleapis.com"],
+    "prj-secure-cloud-function" = ["networksecurity.googleapis.com", "cloudfunctions.googleapis.com", "cloudbuild.googleapis.com", "eventarc.googleapis.com", "eventarcpublishing.googleapis.com"],
```

Comment:
Some of the apis were removed during the update (sqladmin.googleapis.com, cloudscheduler.googleapis.com, servicenetworking.googleapis.com)

Reply:
Oh, thank you. It's fixed now.